File: Pattern1.java

/**
 * Pattern1
 */


public class Pattern1 

{

    public static void Print(int ino)
    {
        for(int i = 1 ; i<=ino ; i++)
        {
            for(int j = 1 ; j<=i ; j++)
            {
                System.out.print(j+" ");
            }
            System.out.println();
        }


    }

    public static void main(String[] args) 
    {
        int ino = 7;

        Print(ino);
        
    }
    
}

File: Pattern1.c
// Input : 7
// Output : # # # # # #  #

// Input : 3
// Output : # # # 

#include<stdio.h>

void Display(int iNo)
{
    int iCnt = 1;

    for(iCnt = 1; iCnt <= iNo; iCnt++)
    {
        printf("#\t");
    }
    printf("\n");
    

}
int main()
{
    int iFrequency = 0;

    printf("ENter the frequency of Symbol : \n");
    scanf("%d",&iFrequency);

    Display(iFrequency);

    return 0;
}